src/components/ShareTrajectoryModal/index.tsx

```diff
@@ -146,148 +136,6 @@ const ShareTrajectoryModal = ({
-    }, [embedSettings]);
-
-    const handleEmbeddedSizeInputs = (
-        setting: "height" | "width",
-        input: string
-    ) => {
-        const numericInput = input ? parseInt(input) : 0;
-        let roundedLimitedInput;
-
-        if (setting === "width") {
-            roundedLimitedInput = numericInput >= 3840 ? 3840 : numericInput;
-        } else if (setting === "height") {
-            roundedLimitedInput = numericInput >= 2160 ? 2160 : numericInput;
-        }
-
-        setEmbedSettings({ ...embedSettings, [setting]: roundedLimitedInput });
-    };
-
-    const EmbedSnippetPanel: JSX.Element = (
-        <VerticalFlexbox gap={8}>
-            <div className={styles.embedHeader}>
-                <h4>Embed &lt;/&gt;</h4>
-                <button
-                    className={styles.settingsButton}
-                    onClick={() => {
-                        setShowEmbedSettingsPanel(!showEmbedSettingsPanel);
-                    }}
-                >
-                    Show settings {DownArrow}
-                </button>
-            </div>
-            {showEmbedSettingsPanel && (
-                <div className={styles.embedSettings}>
-                    <VerticalFlexbox gap={4}>
-                        <p className={styles.accentText}> Size </p>
-                        <VerticalFlexbox gap={24}>
-                            <div className={styles.proportionSettings}>
-                                {" "}
-                                <Input
-                                    disabled={
-                                        !embedSettings.userProvidedProportions
-                                    }
-                                    defaultValue={embedSettings.height}
-                                    className={styles.numberInputs}
-                                    onChange={(e) => {
-                                        handleEmbeddedSizeInputs(
-                                            "height",
-                                            e.target.value
-                                        );
-                                    }}
-                                />
-                                <p> x </p>
-                                <Input
-                                    disabled={
-                                        !embedSettings.userProvidedProportions
-                                    }
-                                    value={embedSettings.width}
-                                    className={styles.numberInputs}
-                                    onChange={(e) => {
-                                        handleEmbeddedSizeInputs(
-                                            "width",
-                                            e.target.value
-                                        );
-                                    }}
-                                />
-                                <p> pixels </p>
```

Comment:
`<p>` should be for an actual paragraph. <span> or <label> are more appropriate semantically here